### PR TITLE
fix: multiple delcaration in same line (#8)

### LINKED ISSUE #8: Wrong Check when multiple declare in same line
1. makezero could make mistake in judge slice append when the slice declare in same line with other non-zero slice:

```
// will also report "append to slice `b` with non-zero initialized length at testing.go:7:9" while it shouldn't
package bar

func foo() {
    a, b := make([]int, 10), make([]int, 0)
    a = append(a, 10)
    b = append(b, 10)
}
```

2. makezero failed to report in judge slice append when the slice declare in same line with other zero slice in front:
```
// will report nothing while it should
package bar

func foo() {
    a, b := make([]int, 0), make([]int, 10)
    a = append(a, 10)
    b = append(b, 10)
}
```

### PR DESCRIPTION
**Pull request by ivila**
Fixes #8

### CODE REVIEW

**Comment by ivila** [comment]
@ashanbrown can you help me to take a look into this pr?

File: makezero/makezero_test.go [hunk 1]
```diff
@@ -79,6 +79,46 @@ func foo() {
 	})
 }
 
+func TestMultiDeclare(t *testing.T) {
+	t.Run("ignores multi declares in same line", func(t *testing.T) {
```
Comment (ashanbrown) [inline makezero/makezero_test.go:83]:
```suggestion
	t.Run("handles multi declares in same line", func(t *testing.T) {
```

File: makezero/makezero_test.go [hunk 2]
```diff
@@ -79,6 +79,46 @@ func foo() {
...
+		t.Run("with first obj is non-zero", func(t *testing.T) {
```
Comment (ashanbrown) [inline makezero/makezero_test.go:84]:
```suggestion
		t.Run("with just first obj is non-zero", func(t *testing.T) {
```

File: makezero/makezero_test.go [hunk 3]
```diff
@@ -79,6 +79,46 @@ func foo() {
...
+package bar
+
+func foo() {
+    a, b := make([]int, 10), make([]int, 0)
+    a = append(a, 10)
+    b = append(b, 10)
+}`, "append to slice `a` with non-zero initialized length at testing.go:6:9")
+		})
+
+		t.Run("with second obj is non-zero", func(t *testing.T) {
```
Comment (ashanbrown) [inline makezero/makezero_test.go:96]:
```suggestion
		t.Run("with just second obj is non-zero", func(t *testing.T) {
```

File: makezero/makezero_test.go [hunk 4]
```diff
@@ -79,6 +79,46 @@ func foo() {
...
+package bar
+
+func foo() {
+    a, b := make([]int, 0), make([]int, 10)
+    a = append(a, 10)
+    b = append(b, 10)
+}`, "append to slice `b` with non-zero initialized length at testing.go:7:9")
+		})
+
+		t.Run("with all obj is non-zero", func(t *testing.T) {
```
Comment (ashanbrown) [inline makezero/makezero_test.go:108]:
```suggestion
		t.Run("with all obj non-zero", func(t *testing.T) {
```

**Review by ashanbrown — APPROVED** [review]
LGTM.  Thank you for your fix!  Made a few small text suggestions but it looks great.

**Pull request merged**